src/_includes/partials/footer.html

```diff
@@ -6,11 +6,16 @@
 		<span class="footer-divider">•</span>
 		<a href="/">Events</a>
 		<span class="footer-divider">•</span>
-		<a href="/sesh">Sesh Help</a>
+		<a href="/sesh">Participate</a>
```

Comment:
I changed this text to say "Participate"

I was thinking maybe we also could repurpose the /sesh page to be /participate or however we see fit to word it and combine both the documentation of the sesh bot in Discord and the CLI method of creating events for the Github repo